src/lib/components/ui/Card/Card.scss

```diff
@@ -7,7 +9,7 @@
 
   display: flex;
   flex-direction: column;
-  width: 100%;
+  min-width: 0; // 1.
```

Comment:
`width` is not necessary and `min-width` should improve behavior in flex and grid layouts.